### ISSUE TITLE
Version of SBDI Sativa curated 16S GTDB database used 

### ISSUE THREAD

**Issue by MaximilianBaumgartner** [borenstein-lab/microbiome-metabolome-curated-data#4]
Thanks a lot for providing this awesome resource !

I would like to merge some of my own 16S data, for this I would need the exact version of the SBDI Sativa 16S database that has been used
(there are Version 1-5, with "1genome", "5genomes" and "20genomes" assignTaxonomy and addSpezies).

https://figshare.scilifelab.se/articles/dataset/SBDI_Sativa_curated_16S_GTDB_database/14869077

Did you use the dada2 assignTaxonomy function with standard parameters and these databases +then remove the species classification or modify the database beforehand and remove the species in the fasta names ?

Cheers,
Max

**Comment by efratmuller** [comment]
Hi Max!

The specific file I used with DADA's `assignTaxonomy` was `gtdb-sbdi-sativa.r07rs207.1genome.assignTaxonomy.fna.gz` from version 5. I now also noted the version number and the file name in the Wiki. 

As for your second question about how I ran the `assignTaxonomy` function, the exact script I used can be found here: https://github.com/borenstein-lab/microbiome-metabolome-curated-data/blob/main/scripts/data_organization/load_original_data/asv_assignTaxonomy_gtdb.R. I basically ran the `assignTaxonomy` function as is, and later slightly edited the format of species names, but I did not remove anything from the sbdi database itself. I hope that answered your question but let me know if it didn't.

Best,
Efrat
